packages/clerk-js/src/core/auth/CaptchaHeartbeat.ts

```diff
@@ -15,10 +15,14 @@ export class CaptchaHeartbeat {
       return;
     }
 
-    await this.challengeAndSend();
-    this.timers.setInterval(() => {
-      void this.challengeAndSend();
-    }, this.intervalInMs());
+    try {
+      await this.challengeAndSend();
+      this.timers.setInterval(() => {
+        void this.challengeAndSend();
+      }, this.intervalInMs());
+    } catch (error) {
```

Comment:
Is the try/catch necessary here considering the challengeAndSend method already has one internally? 🤔

Reply:
Agreed, it's overkill